### ISSUE TITLE
mesh.GetBoundingBox() does not work in parallel

### ISSUE THREAD

**Issue by benzwick** [mfem/PyMFEM#81]
I get the following error:
```
NameError                                 Traceback (most recent call last)
~/projects/EEG/mfem-eeg-python/eeg_leadfield.py in <module>
     60 print("done.")
     61 
---> 62 print("Mesh bounding box: ", mesh.GetBoundingBox())
     63 
     64 # create finite element function space

~/.local/lib/python3.8/site-packages/mfem/_par/pmesh.py in GetBoundingBox(self, ref)
    313         min = Vector()
    314         max = Vector()
--> 315         _mesh.Mesh_GetBoundingBox(self, min, max, ref)
    316         return min.GetDataArray().copy(), max.GetDataArray().copy()
    317 

NameError: name '_mesh' is not defined
```

It looks to me like `_mesh` should be replaced with `_pmesh` here:

https://github.com/mfem/PyMFEM/blob/b3b6b4ada5f7f1dcb70dd53fa17af569711278fd/mfem/_par/pmesh.py#L315

and also here:

https://github.com/mfem/PyMFEM/blob/b3b6b4ada5f7f1dcb70dd53fa17af569711278fd/mfem/_par/pmesh.py#L340

**Comment by sshiraiwa** [comment]
Thanks for reporting this. A fix is being in test...

**Comment by benzwick** [comment]
This is fixed in 4.2.0.16 #84
